apps/api/v2/src/modules/organizations/repositories/organizations-users.repository.ts

```diff
@@ -37,22 +37,6 @@ export class OrganizationsUsersRepository {
     });
   }
 
-  async getOrganizationUserByUsername(orgId: number, username: string) {
```

Comment:
@joeauyeung had added this code but I replaced it with 'getOrganizationUserByUsername' below which does not check 'User.username' like this function did but 'Profile.username'.